packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -381,11 +383,14 @@ describe('Facet Test Suite', () => {
 
   describe('with link values', () => {
     function setupWithLinkValues() {
-      visitFacetPage({
-        field: defaultField,
-        label: defaultLabel,
-        displayValuesAs: 'link',
-      });
+      visitFacetPage(
+        {
+          field: defaultField,
+          label: defaultLabel,
+          displayValuesAs: 'link',
+        },
+        false
```

Comment:
Could you help me understand why we explicitely don't want to wait for search to return in this case?

Reply:
I have expectations that need to wait for search

Reply:
The first assertion in the test seems to assume the search has returned though. Could you show me which one requires it please?
```
Expect.displayPlaceholder(false);
```

Reply:
`    aliasFacetValues();
        Expect.facetValuesEqual(indexFacetValuesAlias);`

` function aliasFacetValues() {
    cy.wait(InterceptAliases.Search).then((interception) => {
      const indexValues = extractFacetValues(interception.response);
      cy.wrap(indexValues).as(indexFacetValuesAlias.substring(1));
    });
  }`